cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,72 @@
 /// <reference types='cypress' />
 
 describe('Web Tables page', () => {
-  it('', () => {
+  beforeEach(() => {
+    cy.visit('/webtables');
+  });
+
+  it('should can change rows count', () => {
```

Comment:
The test description 'should can change rows count' is grammatically incorrect. Consider changing it to 'should change rows count'.